README.md

```diff
@@ -8,7 +8,32 @@ Okta Spring Boot Starter
 
 Okta's Spring Boot Starter will enable your Spring Boot application to work with Okta via OAuth 2.0/OIDC.
 
-**NOTE:** This library works with Spring Boot 2.2+. If you need support for Spring Boot 1.5.x, use version 0.6.
+**NOTE:** This library works with Spring Boot 2.4+. If you need support for Spring Boot 1.5.x, use version 0.6.
+
+## Release status
+
+This library uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/).
+
+:heavy_check_mark: The current stable major version series is: 1.5.x
```

Comment:
We can probably just say `1.x`, it's only the major versions that fall under the "Okta library version policy"